### PR TITLE
docs: update model-serving with mount vfolder and validation

### PR DESCRIPTION
**Pull request by lizable**
related PR: [lablup/backend.ai-webui#2437](https://github.com/lablup/backend.ai-webui/pull/2437)

This PR updates model serving menu with updated UI and features:
- additional mounts on service creation/modification
- arbitrary setting on model storage mount path and model definition file path
- model validation
- rearranging 7.3 and sub-menu

Preview of model serving pages
- [Backend.AI WebUI Docs 24.03.4-model_serving.pdf](https://github.com/user-attachments/files/15813213/Backend.AI.WebUI.Docs.24.03.4-model_serving.pdf)
- [Backend.AI WebUI Docs 24.03.4-model_serving-ko.pdf](https://github.com/user-attachments/files/15813216/Backend.AI.WebUI.Docs.24.03.4-model_serving-ko.pdf)

### CODE REVIEW

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 1]
```diff
@@ -380,17 +380,17 @@ msgstr ""
 msgid "Model definition file upload"
 msgstr "모델 정의 파일 업로드"
 
-#: ../../model_serving/model_serving.rst:196
-msgid "Creating/Modifying Model Service"
-msgstr "모델 서비스 생성하기/수정하기"
+#: ../../model_serving/model_serving.rst:197
+msgid "Creating/Validating Model Service"
+msgstr "모델 서비스 생성하기"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:385]:
```suggestion
msgstr "모델 서비스 생성하기/유효성 검사하기"
```
Or
```suggestion
msgstr "모델 서비스 생성하기/검사하기"
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 2]
```diff
@@ -430,14 +462,13 @@ msgstr ""
...
 "기준으로 서비스하려는 세션을 몇 개까지 만들어야 할지 설정하는데, 이 값을 기준으로 하게 됩니다."
 
-#: ../../model_serving/model_serving.rst:228
+#: ../../model_serving/model_serving.rst:236
 msgid ""
 "Then select a resource group. The resource group is a collection of "
 "resources that can be allocated to the model service."
-msgstr ""
-"그리고, 이미지 환경과 리소스 그룹을 선택합니다. 리소스 그룹은 모델 서비스에 할당될 수 있는 리소스의 집합입니다."
+msgstr "그리고, 이미지 환경과 리소스 그룹을 선택합니다. 리소스 그룹은 모델 서비스에 할당될 수 있는 리소스의 집합입니다."
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:469]:
```suggestion
msgstr "그리고, 이미지 환경과 자원 그룹을 선택합니다. 자원 그룹은 모델 서비스에 할당될 수 있는 자원의 집합입니다."
```
Let's standardize the terminology for resource group to "자원 그룹".

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 3]
```diff
@@ -732,3 +787,20 @@ msgstr ""
...
+#~ "모델 정의 파일은 현재 버전 기준으로 ``model-"
+#~ "definition.yml``, 또는 ``model-definition.yaml`` "
+#~ "로 파일명과 확장자를 고정해야 합니다."
+
+#~ msgid ""
+#~ "First, provide a service name and "
+#~ "select the virtual folder of the "
+#~ "model type to be used for the "
+#~ "model service."
+#~ msgstr "우선 서비스 이름과 모델 서비스에 사용할 모델 타입의 가상 폴더를 선택합니다. "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:805]:
```suggestion
```

File: docs/model_serving/model_serving.rst [hunk 4]
```diff
@@ -203,25 +204,32 @@ bring up a modal where you can enter the required settings for creating
...
-the virtual folder of the model type to be used for the model service.
-
-For detailed explanations of each item, please refer to the following:
+First, provide a service name. For detailed explanations of each item, please refer to the following:
 
 -  Open To Public: This option allows access to the model service
    without any separate token on the server where the service is to be
    hosted. By default, it is disabled.
+-  Model Storage To Mount: This is model folder to mount, which contains 
+   model definition file inside the directory
```
Comment (agatha197) [inline docs/model_serving/model_serving.rst:217]:
```suggestion
   model definition file inside the directory.
```

File: docs/model_serving/model_serving.rst [hunk 5]
```diff
@@ -203,25 +204,32 @@ bring up a modal where you can enter the required settings for creating
...
+-  Model Destination For Model Folder: This option allows aliasing path of 
+   model storage path to session corresponding to routing, which represents 
+   the service. default value is ``/models``.
+-  Model Definition File Path: You can also set model definition file as you
+   uploaded in model storage path. default value is ``model-definition.yaml``.
```
Comment (agatha197) [inline docs/model_serving/model_serving.rst:222]:
```suggestion
   uploaded in the model storage path. The default value is ``model-definition.yaml``.
```

File: docs/model_serving/model_serving.rst [hunk 6]
```diff
@@ -247,14 +255,28 @@ resources that can be allocated to the model service.
...
 -  Single Node : When running a session, the managed node and worker nodes are 
    placed on a single physical node or virtual machine.
 -  Multi Node : When running a session, one managed node and one or more worker 
    nodes are split across multiple physical nodes or virtual machines.
 
+Before creating model service, Backend.AI supports validation feature to check 
+whether execution is available or not(due to any errors during execution.) By 
+clicking ``Validate`` button in bottom-left of service launcher, a new popup for 
+listening validation event will pops up. in the popup modal, you can check the status 
+by container log. when result is set to ``Finished``, then validation check is finished.
```
Comment (agatha197) [inline docs/model_serving/model_serving.rst:270]:
```suggestion
whether execution is available or not(due to any errors during execution.) By clicking the ``Validate`` button at the bottom-left of the service launcher, a new popup for listening to validation events will pop up. In the popup modal, you can check the status through the container log. When the result is set to ``Finished``, then the validation check is finished.
```

**Review by agatha197 — CHANGES_REQUESTED** [review]
I left minor suggestions :)

**Review by agatha197 — CHANGES_REQUESTED** [review]
image
Some sentences end with a period(.), while others do not. Additionally, some end with a noun, while others end with a verb. Could you please make the style consistent throughout the document?

**Comment by lizable** [comment]
> image Some sentences end with a period(.), while others do not. Additionally, some end with a noun, while others end with a verb. Could you please make the style consistent throughout the document?

Okay, good suggestion.

File: docs/model_serving/model_serving.rst [hunk 7]
```diff
@@ -247,14 +255,31 @@ resources that can be allocated to the model service.
    any AI accelerator on demand according to resource group.
 
 .. image:: cluster_mode.png
-   :width: 500
+   :width: 700
    :align: center
 
--  Single Node : When running a session, the managed node and worker nodes are 
+-  Single Node:When running a session, the managed node and worker nodes are
```
Comment (agatha197) [inline docs/model_serving/model_serving.rst:261]:
```suggestion
-  Single Node: When running a session, the managed node and worker nodes are 
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 8]
```diff
@@ -164,13 +164,13 @@ msgstr "모델 정의 파일 생성하기"
 
 #: ../../model_serving/model_serving.rst:77
 msgid ""
-"The model definition file must be named ``model-definition.yml`` or "
-"``model-definition.yaml`` to align with the current version."
+"From 24.03, you can configure model definition file name. But if you "
+"don't input any other input field in model definition file path, then the"
+" system will regard it as ``model-definition.yml`` or ``model-"
+"definition.yaml``"
 msgstr ""
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:171]: translation is missing.

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 9]
```diff
@@ -401,125 +401,177 @@ msgstr "모델 서빙 페이지에서 \"서비스 시작하기\" 버튼을 클
...
 "CPU: The number of CPU cores allocated to the routing for running the "
 "model service."
 msgstr "CPU: 모델 서비스를 실행하는 라우팅, 즉 연산 세션에 할당할 CPU코어의 수."
 
-#: ../../model_serving/model_serving.rst:238
+#: ../../model_serving/model_serving.rst:246
 msgid ""
 "RAM: The amount of memory allocated to the routing for running the model "
 "service (in GiB)."
 msgstr "RAM: 연산 세션에 할당할 메모리의 용량(GiB)"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:489]:
```suggestion
msgstr "RAM: 연산 세션에 할당할 메모리의 용량(GiB)."
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 10]
```diff
@@ -401,125 +401,177 @@ msgstr "모델 서빙 페이지에서 \"서비스 시작하기\" 버튼을 클
...
 
-#: ../../model_serving/model_serving.rst:240
+#: ../../model_serving/model_serving.rst:248
 msgid "GPU: The GPU allocation for the routing for running the model service."
 msgstr "GPU: 연산 세션에 할당할 GPU 단위"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:493]:
```suggestion
msgstr "GPU: 연산 세션에 할당할 GPU 단위."
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 11]
```diff
@@ -401,125 +401,177 @@ msgstr "모델 서빙 페이지에서 \"서비스 시작하기\" 버튼을 클
...
 
-#: ../../model_serving/model_serving.rst:242
+#: ../../model_serving/model_serving.rst:250
 msgid ""
 "Shared Memory: The amount of shared memory allocated to the routing for "
 "running the model service (in GiB). It should be smaller than the "
 "allocated memory."
 msgstr "Shared Memory: 연산 세션에 할당할 공유 메모리의 용량(GiB). 연산 세션에 할당할 메모리의 용량보다는 작아야 함"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:500]:
```suggestion
msgstr "Shared Memory: 연산 세션에 할당할 공유 메모리의 용량(GiB). 연산 세션에 할당할 메모리의 용량보다는 작아야 함."
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 12]
```diff
@@ -401,125 +401,177 @@ msgstr "모델 서빙 페이지에서 \"서비스 시작하기\" 버튼을 클
...
+#: ../../model_serving/model_serving.rst:253
 msgid ""
 "AI Accelerator: The amount of AI accelerator (GPUs or NPUs). If manual "
 "image setting is allowed by configuration, users could select any AI "
 "accelerator on demand according to resource group."
 msgstr ""
-"AI Accelerator : AI 가속기 (GPUs 또는 NPUs)의 양입니다. 설정에 의해 수동 이미지 설정이 허용된 경우, "
-"사용자는 선택한 리소스 그룹에 따라 AI 가속기를 선택할 수 있습니다."
+"AI Accelerator: AI 가속기 (GPUs 또는 NPUs)의 용량으로 설정에 의해 수동 이미지 설정이 허용된 경우, 선택한"
+" 자원 그룹에 따라 AI 가속기를 선택하게 됨"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:509]:
```suggestion
" 자원 그룹에 따라 AI 가속기를 선택하게 됨."
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 13]
```diff
@@ -401,125 +401,177 @@ msgstr "모델 서빙 페이지에서 \"서비스 시작하기\" 버튼을 클
...
-#: ../../model_serving/model_serving.rst:253
+#: ../../model_serving/model_serving.rst:261
 msgid ""
-"Single Node : When running a session, the managed node and worker nodes "
-"are placed on a single physical node or virtual machine."
-msgstr ""
-"Single Node : 세션을 실행할 때에 관리 노드와 워커 노드들이 하나의 물리 노드 또는 가상 머신에 배치하는 경우입니다."
+"Single Node:When running a session, the managed node and worker nodes are"
+" placed on a single physical node or virtual machine."
+msgstr "Single Node: 세션을 실행할 때에 관리 노드와 워커 노드들이 하나의 물리 노드 또는 가상 머신에 배치되는 경우"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:515]:
```suggestion
msgstr "Single Node: 세션을 실행할 때에 관리 노드와 워커 노드들이 하나의 물리 노드 또는 가상 머신에 배치되는 경우."
```

**Review by agatha197 — CHANGES_REQUESTED** [review]

**Review by YEONFEEL96 — CHANGES_REQUESTED** [review]
KR/EN Reviewed

File: docs/locale/ko/LC_MESSAGES/import_run/import_run.po [hunk 14]
```diff
@@ -193,32 +189,18 @@ msgstr ""
...
+#~ msgstr "Jupyter notebook을 바로 실행할 수 있는 버튼 생성하기"
+
+#~ msgid "Import GitHub repository"
+#~ msgstr "GitHub 저장소 불러오기"
+
+#~ msgid "Import GitHub repository result"
+#~ msgstr "GitHub 저장소 불러오기"
+
+#~ msgid "Import GitLab repository"
+#~ msgstr "GitLab 저장소 불러오기"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/import_run/import_run.po:205]:
```suggestion
```

File: docs/locale/ko/LC_MESSAGES/login/login.po [hunk 15]
```diff
@@ -190,108 +190,6 @@ msgid ""
...
-#~ " sidebar narrower. Click again to "
-#~ "bring the sidebar back to its "
-#~ "original width."
-#~ msgstr ""
-#~ "가장 왼쪽의 버튼은 사이드바의 크기를 변경합니다. 클릭하면"
-#~ " 사이드바의 폭이 크게 줄어들면서 내용을 보다 넓게"
-#~ " 볼 수 있습니다. 다시 한 번 클릭하면 "
-#~ "사이드바의 폭이 다시 원래대로 돌아옵 니다."
+#~ msgid "Signup dialog"
+#~ msgstr "가입 및 로그인"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/login/login.po:194]:
```suggestion
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 16]
```diff
@@ -188,115 +199,115 @@ msgstr ""
...
-#: ../../model_serving/model_serving.rst:90
+#: ../../model_serving/model_serving.rst:93
 msgid "The model definition file follows the following format:"
 msgstr "모델 정의 파일은 다음과 같은 형식을 따릅니다:"
 
-#: ../../model_serving/model_serving.rst:113
-msgid "Key-Value Descriptions for Model Definition File"
+#: ../../model_serving/model_serving.rst:115
+msgid "**Key-Value Descriptions for Model Definition File**"
 msgstr "모델 정의 파일에 대한 키-값 설명"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:208]:
```suggestion
msgstr "**모델 정의 파일에 대한 키-값 설명**"
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 17]
```diff
@@ -378,207 +389,215 @@ msgstr ""
...
-msgid "Terminate model service dialog"
-msgstr "모델 서비스 종료하기"
+"``Finished`` 결과 값이 실행이 성공적으로 종료되었음을 의미하지는 않습니다. 그 대신, 컨테이너 로그를 반드시 확인하시기 "
+"바랍니다."
 
-#: ../../model_serving/model_serving.rst:289
-msgid "Handling Failed Model Service Creation"
+#: ../../model_serving/model_serving.rst:284
+msgid "**Handling Failed Model Service Creation**"
 msgstr "모델 서비스 생성에 실패한 경우"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:564]:
```suggestion
msgstr "**모델 서비스 생성에 실패한 경우**"
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 18]
```diff
@@ -706,29 +725,48 @@ msgstr ""
...
+#: ../../model_serving/model_serving.rst:385
+msgid ""
+"Click on the wrench icon in the Control tab to open a modal where you can"
+" change the model service. The format is identical to the model service "
+"start modal, with previously entered fields already filled in. You can "
+"optionally modify only the fields you wish to change. After modifying the"
+" fields, click the confirm button. The changes will be adjusted "
+"accordingly."
+msgstr ""
+"제어 탭에서 톱니바퀴 아이콘을 클릭하면 원하는 세션 수 값을 변경할 수 있는 모달이 뜨게 됩니다. 수정 모달은 모델 서비스를 "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:741]:
```suggestion
"제어 탭에서 '설정' 버튼을 클릭하면 원하는 세션 수 값을 변경할 수 있는 모달이 뜨게 됩니다. 수정 모달은 모델 서비스를 "
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 19]
```diff
@@ -706,29 +725,48 @@ msgstr ""
...
+"the Backend.AI scheduler. Therefore, it is recommended to terminate the "
+"model service if it is no longer needed. To terminate the model service, "
+"click on the trash icon in the Control tab. A modal will appear asking "
+"for confirmation to terminate the model service. Clicking ``OK`` will "
+"terminate the model service. The terminated model service will be removed"
+" from the list of model services."
+msgstr ""
+"모델 서비스는 주기적으로 스케줄러를 실행하여, 원하는 세션 수와 실제 대응하는 라우팅 수가 원하는 세션 수에 맞춰지도록 "
+"스케줄링합니다. 다만, 이 경우 Backend.AI 스케줄러에 부하가 가는 것은 불가피 합니다. 따라서 모델 서비스를 더 이상 "
+"사용하지 않는 경우라면, 모델 서비스를 종료하는 것이 좋습니다. 모델 서비스를 종료하려면, 제어 탭에서 휴지통 아이콘을 클릭합니다."
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:766]:
```suggestion
"사용하지 않는 경우라면, 모델 서비스를 종료하는 것이 좋습니다. 모델 서비스를 종료하려면, 제어 탭에서 '휴지통' 버튼을 클릭합니다."
```

File: docs/locale/ko/LC_MESSAGES/mount_vfolder/mount_vfolder.po [hunk 20]
```diff
@@ -253,61 +253,6 @@ msgstr ""
...
-#~ "/install-user-programs.rst>`_."
-#~ msgstr ""
-#~ "위와 같은 방식으로 자동 마운트 폴더를 이용하여 "
-#~ "다양한 설정 관리가 가능합니다. 보다 자세한 사항은 "
-#~ "`Backend.AI 위키 "
-#~ "<https://github.com/lablup/backend.ai/blob/master/docs/install"
-#~ "/install-user-programs.rst>`_ 에서 확인할 수 "
-#~ "있습니다."
+#~ msgid "Launch a compute session with storage folders"
+#~ msgstr "자동 마운트 폴더를 이용하여 연산 세션 환경 구성하기"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/mount_vfolder/mount_vfolder.po:257]:
```suggestion
```

File: docs/locale/ko/LC_MESSAGES/sessions_all/sessions_all.po [hunk 21]
```diff
@@ -1231,3 +1231,18 @@ msgid ""
...
+
+#~ msgid "Select customized image"
+#~ msgstr "사용자 정의 이미지 선택하기"
+
+#~ msgid "Copy customized image"
+#~ msgstr "사용자 정의 이미지 복사하기"
+
+#~ msgid "Session HPC Optimization"
+#~ msgstr "세션 HPC 최적화하기"
+
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/sessions_all/sessions_all.po:1248]:
```suggestion
```

File: docs/locale/ko/LC_MESSAGES/sessions_all/sessions_all.po [hunk 22]
```diff
@@ -966,12 +966,12 @@ msgstr ""
 "이름을 입력합니다. 세션 이름은 알파벳 또는 숫자, 또는 하이픈(``-``) 이나 언더바(``_``)로 구성할 수 있으며, 최소 "
 "4글자에서 32자까지만 입력해야 합니다. 세션 이름을 입력한 뒤에는 ``PUSH SESSION TO CUSTOMIZED "
 "IMAGE`` 버튼을클릭합니다. 이렇게 만들어진 사용자 정의 이미지는 추후 세션 생성 시 사용할 수 있습니다. 단, 이미지 커밋을 "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/sessions_all/sessions_all.po:968]:
```suggestion
"IMAGE' 버튼 을클릭합니다. 이렇게 만들어진 사용자 정의 이미지는 추후 세션 생성 시 사용할 수 있습니다. 단, 이미지 커밋을 "
```

File: docs/locale/ko/LC_MESSAGES/sessions_all/sessions_all.po [hunk 23]
```diff
@@ -966,12 +966,12 @@ msgstr ""
 "이름을 입력합니다. 세션 이름은 알파벳 또는 숫자, 또는 하이픈(``-``) 이나 언더바(``_``)로 구성할 수 있으며, 최소 "
 "4글자에서 32자까지만 입력해야 합니다. 세션 이름을 입력한 뒤에는 ``PUSH SESSION TO CUSTOMIZED "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/sessions_all/sessions_all.po:967]:
```suggestion
"4글자에서 32자까지만 입력해야 합니다. 세션 이름을 입력한 뒤에는 'PUSH SESSION TO CUSTOMIZED "
```

File: docs/locale/ko/LC_MESSAGES/sftp_to_container/sftp_to_container.po [hunk 24]
```diff
@@ -618,23 +614,9 @@ msgid ""
...
-#~ "버전과 호환되는 Web-UI 앱을 받아야 최상의 "
-#~ "호환성을 유지할 수 있습니다. Backend.AI 클러스터의 "
-#~ "버전은 GUI 우측 상단에 있는 설정 아이콘을 "
-#~ "클릭하면 나오는 “About Backend.AI” 메뉴를 클릭하여 "
-#~ "확인할 수 있습니다."
+#~ msgid "Starting SSH/SFTP daemon inside a compute session (container)"
+#~ msgstr "연산 세션에 SSH/SFTP 접속하기 (Linux/Mac/Visual Studio Code)"
+
+#~ msgid "Filezilla settings to connect to compute session"
+#~ msgstr "연산 세션에 SSH/SFTP 접속하기 (Linux/Mac/Visual Studio Code)"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/sftp_to_container/sftp_to_container.po:621]:
```suggestion
```

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 25]
```diff
@@ -116,8 +116,8 @@ msgid ""
 "setting, please refer to the :ref:`Quota Setting Panel<quota-setting-"
 "panel>` section."
 msgstr ""
-"쿼타(가용량)는 쿼타 설정을 제공하는 스토리지에서만 사용할 수 있습니다(XFS, CephFS, NetApp, Purestorage 등). 가용량 설정에 "
-"대해서는 :ref:`쿼타 설정 패널<quota-setting-panel>` 섹션을 참고하십시오."
+"쿼타(가용량)는 쿼타 설정을 제공하는 저장소에서만 사용할 수 있습니다(XFS, CephFS, NetApp, Purestorage "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:119]:
```suggestion
"쿼타(가용량)는 쿼타 설정을 제공하는 저장소에서만 사용할 수 있습니다. (XFS, CephFS, NetApp, Purestorage "
```

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 26]
```diff
@@ -116,8 +116,8 @@ msgid ""
...
+"등). 가용량 설정에 대해서는 :ref:`쿼타 설정 패널<quota-setting-panel>` 섹션을 참고하십시오."
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:120]:
```suggestion
"등) 가용량 설정에 대해서는 :ref:`쿼타 설정 패널<quota-setting-panel>` 섹션을 참고하십시오."
```

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 27]
```diff
@@ -177,8 +177,8 @@ msgid ""
 "NFS hosts. You can check whether the selected host has enough capacity "
 "remaining through the indicator."
 msgstr ""
-"Host: 폴더를 만들 NFS 호스트입니다. NFS 호스트가 여러 개인 경우 하나를 선택할 수 있습니다."
-"선택한 호스트가 충분한 용량을 가지고 있는지 여부는 표시기를 통해 확인할 수 있습니다."
+"Host: 폴더를 만들 NFS 호스트입니다. NFS 호스트가 여러 개인 경우 하나를 선택할 수 있습니다.선택한 호스트가 충분한 "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:180]:
```suggestion
"Host: 폴더를 만들 NFS 호스트입니다. NFS 호스트가 여러 개인 경우 하나를 선택할 수 있습니다. 선택한 호스트가 충분한 "
```

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 28]
```diff
@@ -367,7 +367,7 @@ msgstr ""
 "리스트를 볼 수 있습니다. 폴더를 생성할 때 점(``.``)으로 시작하는 이름을 지정하면 Folders 탭이 아닌 Automount"
 " Folders 탭에 추가됩니다. Automount Folders는 연산 세션 생성 시 별도로 마운트 하지 않더라도 자동으로 홈 "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:368]:
```suggestion
"리스트를 볼 수 있습니다. 폴더를 생성할 때 점(``.``)으로 시작하는 이름을 지정하면 'Folders' 탭이 아닌 'Automount"
" Folders' 탭에 추가됩니다. 자동 마운트 폴더는 연산 세션 생성 시 별도로 마운트 하지 않더라도 자동으로 홈 "
```

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po [hunk 29]
```diff
@@ -717,3 +719,49 @@ msgstr ""
...
+
+#~ msgid "Folder explorer with SFTP Server"
+#~ msgstr "SFTP 서버와 폴더 탐색기"
+
+#~ msgid "SSH / SFTP connection dialog"
+#~ msgstr "SSH / SFTP 연결 다이얼로그"
+
+#~ msgid "SSH / SFTP connection dialog expanded"
+#~ msgstr "확장된 SSH / SFTP 연결 다이얼로그"
+
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:767]:
```suggestion
```

**Review by agatha197 — CHANGES_REQUESTED** [review]

File: docs/locale/ko/LC_MESSAGES/import_run/import_run.po (hunk 14 above)
Comment (lizable) [reply docs/locale/ko/LC_MESSAGES/import_run/import_run.po:205]: Let's apply these things as an separate issue(and PR) to split the "interests"(관심사 분리를 위해 별도 이슈로 다룹시다.)

File: docs/locale/ko/LC_MESSAGES/login/login.po (hunk 15 above)
Comment (lizable) [reply docs/locale/ko/LC_MESSAGES/login/login.po:194]: Same here. PR gets too big.

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po (hunk 25 above)
Comment (lizable) [reply docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:119]: This as well.

File: docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po (hunk 26 above)
Comment (lizable) [reply docs/locale/ko/LC_MESSAGES/vfolder/vfolder.po:120]: Here as well.

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**